lib/wicked/wlan.pm

```diff
@@ -0,0 +1,163 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Base class for all WLAN related tests
+# Maintainer: cfamullaconrad@suse.com
+
+
+package wicked::wlan;
```

Comment:
I would say that you gather in this file "everyhting what I need to test WLAN" . Which might be not the best approach , some of this functions would looks better in wickedbase , some of them might be candidates for dedicated file/class ( ahhh I used this word in perl context )

Reply:
Yes that is the purpose!